Packs/MicrosoftSafeLinksStatistics/Integrations/MicrosoftSafeLinksStatistics/README.md

```diff
@@ -0,0 +1,132 @@
+Use the integration to get statistics from the O365 service on Safe Links clicks
+This integration was integrated and tested with version xx of Microsoft SafeLinks Statistics
+## Configure Microsoft SafeLinks Statistics on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Microsoft SafeLinks Statistics.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Required** |
+    | --- | --- |
+    | Exchange Online URL | True |
+    | Email (Password for basic authentication only) | False |
+    | Trust any certificate (not secure) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### o365-safelinks-stats-auth-start
+***
+Starts the OAuth2.0 authorization process.
+
+
+#### Base Command
+
+`o365-safelinks-stats-auth-start`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command Example
+``` ```
```

Comment:
Currently all examples are empty in the README
We can either remove them completely or you can fill them in, what do you prefer to do ?

Reply:
Lets remove all of them.